### ISSUE TITLE
Please update Brunch version 111 stable

### ISSUE THREAD

**Issue by liangcepaul** [sebanc/brunch#1806]
I need it.

**Comment by tejasraman** [comment]
@liangcepaul Brunch 110 will work with ChromeOS 111

**Comment by liangcepaul** [comment]
Stuck at logo.

**Comment by tejasraman** [comment]
@liangcepaul running 110 stable with 111 now, it works fine. What's your laptop and recovery?

**Comment by liangcepaul** [comment]
ThinkPad E460

**Comment by tejasraman** [comment]
Can you try reinstalling brunch with r110 stable and Rammus?

**Comment by liangcepaul** [comment]
r110 not working.

**Comment by liangcepaul** [comment]
I'm using Rammus V107, it works fine.

**Comment by tejasraman** [comment]
Ok, could you please install 110 and enable verbose mode in brunch settings?

**Comment by liangcepaul** [comment]
How to enable verbose mode?

**Comment by liangcepaul** [comment]
It executes a bunch of instructions. How to troubleshoot?

**Comment by tejasraman** [comment]
You just keep clicking enter until you see verbose.

Take a picture when it gets stuck and send it to me.

**Comment by liangcepaul** [comment]
How do I get into ChromeOS?

**Comment by tejasraman** [comment]
On the bootloader screen you can see the options, right?

**Comment by liangcepaul** [comment]
Yes, but after the verbose mode, I can see the same chrome OS Logo on my screen. 
![Uploading PXL_20230321_034544395.jpg…]()

**Comment by tejasraman** [comment]
Could you try brunch 110 stable with the **nami** recovery?

**Comment by liangcepaul** [comment]
What's the nami recovery?

**Comment by tejasraman** [comment]
Another recovery image.

Unlike macOS or Windows, chromeOS has many different images for hardware variations. Nami is a different image than rammus, etc.

You can get it from [https://cros.tech/device/nami](https://CrOS.tech/device/nami)

**Comment by liangcepaul** [comment]
I'm using brunch 110 stable binary image, and I update with the r107 version ChromeOS.

**Comment by tejasraman** [comment]
So can you try a newer version of ChromeOS than 107 (e.g 111?)

**Comment by liangcepaul** [comment]
I tried a few times, first I was using r111, same for r110 and r109.

**Comment by tejasraman** [comment]
Can you try the nami 110 image with brunch 110? Can you reinstall it from windows/Linux?

**Comment by liangcepaul** [comment]
Sure, I'm using the Linux mint, and I could only try it.

**Comment by tejasraman** [comment]
Thanks so much @liangcepaul

**Comment by liangcepaul** [comment]
I have already try installing the Nami Recovery image, I see the same issue, the ChromeOS Logo here.

**Comment by liangcepaul** [comment]
I look the sound button and the microphone button. They are disabled.
